### ISSUE TITLE
import error in python 3.5

### ISSUE THREAD

**Issue by wannaphongcom** [Suyash458/WiktionaryParser#3]
I using
```python
>>> from wiktionaryparser import WiktionaryParser
Traceback (most recent call last):
  File "<pyshell#10>", line 1, in <module>
    from wiktionaryparser import WiktionaryParser
  File "C:\WinPython-64bit-35\python-3.5.1.amd64\lib\site-packages\wiktionaryparser\__init__.py", line 1, in <module>
    from WikiParse import WiktionaryParser
ImportError: No module named 'WikiParse'
```
WiktionaryParser not support python 3.5 ?

**Comment by Suyash458** [comment]
Doesn't support Python 3.5 yet

**Comment by C0rn3j** [comment]
Was this really fixed? Getting an error on both Python 3.5 and 3.6

```
[0] % python
Python 3.6.5 (default, May 11 2018, 04:00:52) 
[GCC 8.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from wiktionaryparser import WiktionaryParser
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'WiktionaryParser'
```
